.kitchen.cloud.yml

```diff
@@ -14,8 +14,7 @@ driver_config:
     - <%= ENV['OS_SECURITY_GROUP_NO_FIREWALL'] %>
 
 provisioner:
-  name: chef_solo
-  attributes:
+  name: chef_zero
```

Comment:
I thought Chef Zero was required for Berkshelf. After some testing, I found this is false. I'm happy to change this back.